src/table/components/__tests__/TableBodyWrapper.spec.jsx

```diff
@@ -44,8 +44,17 @@ describe('<TableBodyWrapper />', () => {
 
   beforeEach(async () => {
     jest.spyOn(selectionsUtils, 'addSelectionListeners').mockImplementation(() => jest.fn());
+    const getLayout = generateLayout(1, 1, 2);
+    getLayout.totals = {
+      label: 'Totals',
+    };
+    getLayout.qHyperCube.qGrandTotalRow = [
+      {
+        qText: '100',
+      },
+    ];
```

Comment:
I see you adding this to at least two test files. Does it work to add this to generateLayout?